lib/modules/manager/npm/extract/__snapshots__/index.spec.ts.snap

```diff
@@ -525,6 +525,9 @@ exports[`modules/manager/npm/extract/index > .extractPackageFile() > extracts vo
   ],
   "extractedConstraints": {
     "node": "8.9.2",
+    "npm": "5.9.0",
+    "pnpm": "6.11.2",
+    "yarn": "1.12.3",
```

Comment:
What's the benefit of extracting these?
- I don't think they are used during lookups (e.g. for compatibility like node is)
- If we did use them in post-upgrades then we should extract them from the latest/current version of the file, because they may have been updated

It might be better that these ones which are only used in post-update steps into `managerData`

Reply:
1. Yes, they aren't used during lookups
2. We do extract them during post-update as use those values. This extraction here is redundant